src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.td

```diff
@@ -616,15 +619,16 @@ def replaceONNXLSTMPattern1 : Pattern<
       $hidden_bias,                                      // hidden bias
       $hidden_size,                                      // hidden_size
       $direction,                                        // direction
-      (GetLSTMGRUReturnAllStepsAttr $res__0, $res__1)),  // return_all_steps
+      (GetLSTMGRUReturnAllStepsAttr $res__0, $res__1),   // return_all_steps
+      (GetStrAttr<"none">)),                             // prev_layer
 
    //
    // Post-process to generate three return values of ONNXLSTMOp
    // from two return values of ZHighLSTMOp(=$zHighLSTM).
    //
    (ZHighUnstickOp:$unstick_HnOutput $zhighLSTM__0, (returnType $res__0)),
    (GetLSTMGRUGetY $unstick_HnOutput, $res__0),
-   (GetLSTMGRUGetYh $unstick_HnOutput, $res__0, $res__1, $direction),
+   (GetLSTMGRUGetYh $unstick_HnOutput, $res__0, $res__1, (GetShape $x), $direction),
```

Comment:
I would pass `$x` instead of `GetShape $x` to be consistent with the other arguments. The same for other places.

Reply:
Updated. Thank you.